int32 object_type
float64 angle
float64 distance
